py/dml/expr.py

```diff
@@ -80,8 +84,38 @@ def coverity_markers(markers, site=None):
         site_linemark_nocoverity(site)
 
 
+# Allow linemarks to be generated if dml.globals.linemarks_enabled is True
+# and the current output is FileOutput
+# This context manager generates a line directive reset when left unless
+# linemarks were already allowed, or manual_reset is True
+@contextmanager
+def allow_linemarks(manual_reset=False):
```

Comment:
looks like `manual_reset` is never used, so that can be removed.

Reply:
Yeah; I thought I'd need it once I began writing the PR, but nope. Now it just exists as a "just in case" but in this case I agree we can add it later if need be.